content/02.results.md

```diff
@@ -78,11 +92,30 @@ Using cut offs based on agreement with the established PCR marker, 368 individua
 
```

Comment:
as we haven't mentioned arabiensis yet, I think we might need a sentence here just to explain that we think three species are represented in these collections, before we explain how we determine species.